src/index.ts

```diff
@@ -1,5 +1,5 @@
 import FakeXMLHttpRequest from 'fake-xml-http-request';
-import * as FakeFetch from 'whatwg-fetch';
+import * as FakeFetch from './whatwg-fetch';
```

Comment:
You do not need to copy the whole file, `external: Object.keys(pkg.dependencies),` in rollup config is treating whatwg-fetch as external module, when you remove it from `pkg.dependencies` it will be imported by rollup.
You will still need `whatwg-fetch` in dev dependencies to let rollup find the file.

Reply:
Yah. But it would make the next step easier since we can wrap it in a check to see if window exists. Otherwise we’ll need to do what I did in the other PR and bring in cross fetch, I think.